src/main/java/uk/ac/ebi/ena/webin/cli/service/SampleService.java

```diff
@@ -42,15 +42,27 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SampleService.class);
 
-    private final BiosamplesService biosamplesService = new BiosamplesService();
+    private final String biosamplesWebinUserName;
+    private final String biosamplesWebinPassword;
+
+    private final WebinAuthClientService webinAuthClientService;
+
+    private final BiosamplesService biosamplesService;
 
     private final SampleXmlService sampleXmlService;
 
     protected 
-    SampleService( AbstractBuilder<SampleService> builder )
+    SampleService( Builder builder )
     {
         super( builder );
 
+        this.biosamplesWebinUserName = builder.biosamplesWebinUserName;
+        this.biosamplesWebinPassword = builder.biosamplesWebinPassword;
+
+        webinAuthClientService = createWebinAuthClientService();

```

Comment:
If we only use the biosamplesWebinUserName/Password in createWebinAuthClientService() then pass the values to the method and remove the instance variables.